v2/src/modules/txs/ModuleTxDetails.vue

```diff
@@ -1,56 +1,201 @@
+                    <div class="skeleton-box rounded-xl" style="height: 24px; width: 100px"></div>
                 </template>
-            </app-details-list>
-        </v-col>
-    </v-row>
+                <app-chip v-else :bg="titleBg" rounded="xl">
+                    <div class="d-flex align-center">
+                        <p>{{ titleStatus }}</p>
+                        <v-progress-circular v-if="txStatus === TxStatus.pending" indeterminate color="white" size="15" width="3" class="ml-2" />
+                    </div>
+                </app-chip>
+            </div>
+            <v-row class="mt-5">
+                <v-col cols="12" lg="6">
+                    <div class="tx-info">
+                        <p class="text-button mb-1">Tx Hash</p>
+                        <template v-if="loadingTransactionHash">
+                            <div class="skeleton-box rounded-xl mt-1 my-4" style="height: 24px"></div>
+                        </template>
+                        <template v-else>
+                            <app-transform-hash is-blue :hash="transactionData.hash" class="text-body-1" />
+                            <p v-if="txStatus !== TxStatus.pending && !isReplaced" class="text-body-2 text-info mt-1">({{ timestamp }})</p>
+                        </template>
+                    </div>
+                </v-col>
+                <template v-if="txStatus !== TxStatus.pending && !isReplaced">
+                    <v-col cols="12" lg="2">
+                        <div class="tx-info">
+                            <p class="text-button mb-1">Block Mined</p>
+                            <template v-if="loadingTransactionHash">
+                                <div class="skeleton-box rounded-xl mt-1 my-4" style="height: 24px"></div>
+                            </template>
+                            <p v-else class="text-body-1 text-secondary mt-1">{{ formatNumber(blockMined) }}</p>
+                        </div>
+                    </v-col>
+                    <v-col cols="12" lg="2">
+                        <div class="tx-info">
+                            <p class="text-button mb-1">Confirmations</p>
+                            <p v-if="!loadingBlockInfo" class="text-body-1 text-secondary mt-1">{{ confirmations }}</p>
+                            <div v-else class="skeleton-box rounded-xl mt-1" style="height: 24px"></div>
+                        </div>
+                    </v-col>
+                </template>
+            </v-row>
+            <v-row align="center" class="mt-5">
+                <v-col cols="12" lg="5">
+                    <div class="rounded-lg bg-tableGrey pa-6">
+                        <div class="tx-info">
+                            <p class="text-button mb-1">From</p>
+                            <template v-if="loadingTransactionHash">
+                                <div class="skeleton-box rounded-xl mt-1 my-4" style="height: 24px"></div>
```

Comment:
the ui is jumping a bit when going from loading to loaded. It's really hard to catch but i think it's cause the loading bar in  from/to guys is a bit smaller then the hash/identicon container. i think its about 32px. (same in to guy)